magda-storage-api/src/GoogleCloudStorageClient.ts

```diff
@@ -0,0 +1,54 @@
+import { Bucket, Storage } from "@google-cloud/storage";
```

Comment:
If we've got minio do we still need a separate Google Cloud client?

Reply:
I don't think so. Removed Google Cloud client.